database/migrations/2016_09_26_093633_document.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class Document extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('models', function (Blueprint $table) {
```

Comment:
Well, I don't want to cause a problem where someone will have to go back to expand this functionality, but I'm not to a point yet where I know how to get around that :-/ 

Only way I could think of it working is by manipulating the table to accept arrays or so, so we'd have doc[0] doc[1], but that's...the wrong way to do it still,  just not sure how to get around that.